app.py

```diff
@@ -81,47 +82,52 @@ def jacket():
 
 @app.route('/login/', methods=['GET', 'POST'])
 def login():
-    if request.method == 'POST':
-        if is_valid_form_fields(request):
-            flash('Заполните форму!', 'danger')
-            return redirect(url_for('login'))
-
-        user_email = request.form.get('email')
-        password = request.form.get('password')
+    form = LoginForm()
+    if request.method == 'POST' and form.validate():
+        data_user = Users.query.filter_by(user_login=form.user_login.data).first()
+        password = hashlib.sha256(form.password.data.encode('utf-8')).hexdigest()
         # аутентификация пользователя
-        session['email'] = request.form.get('email')
-        session['firstname'] = request.form.get('email').split('@')[0]
-        return redirect(url_for('home'))
+        if data_user and password == data_user.password:
 
+            # Нужно понять что хранить в открытой ссесии !!!!!!!!!!!!
+            session['user_login'] = data_user.user_login
+            return redirect(url_for('home'))
+        else:
+            flash('Пройдите регистрацию', 'warning')
+            return redirect(url_for('login'))
     context = {
         'title': 'Страница входа'
     }
-    return render_template('login.html', **context)
+    return render_template('login.html', **context, form=form)
 
 
 @app.route('/register/', methods=['GET', 'POST'])
 def register():
-    if request.method == 'POST':
-        if is_valid_form_fields(request):
-            flash('Заполните форму!', 'danger')
-            return redirect(url_for('register'))
-
-        # Добавление данных с формы в сессию
-        for name in ['firstname', 'lastname', 'user_email', 'password']:
-            session[name] = request.form.get(name)
-
-        flash('Вы авторизованы!', 'success')
+    form = RegistrationForm()
+
+    if request.method == 'POST' and form.validate():
```

Comment:
И тут проверять будет проще 
```suggestion
    if form.validate_on_submit():
```